core/domain/html_validation_service_test.py

```diff
@@ -29,8 +29,10 @@ class ContentMigrationTests(test_utils.GenericTestBase):
     strings to valid RTE format.
     """
     EXP_ID = 'eid'
+    FILENAME = 'abc.png'
```

Comment:
In a previous comment I suggested inlining these. They don't need to be global to the test class. Please fix that.

Reply:
Sorry. Done.